src/main/java/com/iota/iri/service/dto/GetNeighborsResponse.java

```diff
@@ -58,16 +56,17 @@ public static AbstractResponse create(final List<com.iota.iri.network.Neighbor>
      * A plain DTO of an iota neighbor.
      * 
      */
+    @SuppressWarnings("unused")
     static class Neighbor {
 
         private String address;
-        public long numberOfAllTransactions,
-                numberOfRandomTransactionRequests,
-                numberOfNewTransactions,
-                numberOfInvalidTransactions,
-                numberOfStaleTransactions,
-                numberOfSentTransactions;
-        public String connectionType;
+        long numberOfAllTransactions;
+        long numberOfRandomTransactionRequests;
+        long numberOfNewTransactions;
+        long numberOfInvalidTransactions;
+        long numberOfStaleTransactions;
+        long numberOfSentTransactions;
+        String connectionType;
```

Comment:
If you decided to fix the modifiers why did you choose `default` visibility and not `private`?

Reply:
good point. fixed. can you add the rule to the local pmd file as well?